assets/app/styles/_components.less

```diff
@@ -202,32 +201,64 @@
       .osc-object-highlight();
       .flex(@columns: 1 1 auto);
       max-width: 450px;
+      min-width: 90px;
       border-radius: @border-radius-md;
-      border: 1px solid transparent;
-      // box-shadow: @osc-object-box-shadow; <- remove if it's not used in highlight
+      border: 1px solid #fff;
       padding: 10px;
       position: relative;
       margin: 0 10px 10px 0;
       text-align: center;
-      background-color: @pod-pending-bg;
+      background-color: #fcfcfc;
+      background-image: repeating-linear-gradient(-45deg, transparent, transparent 6px, #eeeeee 6px, #eeeeee 12px);
       -webkit-font-smoothing: antialiased;
+      strong {
+        &:before, &:after {
+          font-family: 'PatternFlyIcons-webfont';
+          font-weight: normal;
+        }
+      }
       &.pod-running {
         background-color: @pod-running-bg;
+        background-image: none;
         .spinner {
           display: none;
         }
       }
-      &.pod-failed, &.pod-warning {
+      &.pod-failed {
```

Comment:
I'd probably pull the shared styles between pod-failed and pod-warning into one common block so we're not repeating them.